first_hour_on_terra/workspace_resource_examples.ipynb

```diff
@@ -0,0 +1,1552 @@
+    "\n",
+    "    def add_gcs_bucket(self, b):\n",
+    "        with self.output:\n",
+    "            description_content = f\"\\\"{self.input_description.value}\\\"\"\n",
+    "\n",
+    "            terraCommand = f\"\"\"terra resource add-ref gcs-bucket \\\\\n",
+    "            --cloning={self.cloning_drop_down.value} \\\\\n",
+    "            --bucket-name={self.input_bucket_name.value} \\\\\n",
+    "            --description={description_content} \\\\\n",
+    "            --name={self.input_name.value} \\\\\n",
+    "            --workspace={CURRENT_WORKSPACE_ID}\n",
+    "            \"\"\"\n",
+    "            print(terraCommand)\n",
+    "            result = subprocess.run([\"terra\", \"resource\", \"add-ref\", \"gcs-bucket\",\n",
+    "                                     f\"--cloning={self.cloning_drop_down.value}\",\n",
+    "                                     f\"--bucket-name={self.input_bucket_name.value}\",\n",
+    "                                     f\"--name={self.input_name.value}\",\n",
+    "                                     f\"--description={description_content}\",\n",
+    "                                     f\"--workspace={CURRENT_WORKSPACE_ID}\"], capture_output=True, text=True)\n",
+    "            print(result.stderr) if not result.stdout else print(result.stdout)\n",
+    "\n",
+    "\n",
+    "# Instantiate widget\n",
+    "add_gcs_bucket_widget = AddGcsBucketWidget()\n",
+    "display(add_gcs_bucket_widget.vb)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 5. Add Google Cloud Storage object\n",
+    "\n",
+    "Run the cell below to create a widget, then populate the widget's input fields and click the button to add a Google Cloud Storage bucket object as a referenced resource in your workspace.\n",
+    "\n",
+    "Widget input parameters include:\n",
+    "\n",
+    "- `Cloning`: _You cannot change the workspace ID after workspace creation._\n",
+    "- `Bucket Name`: Must be a string that [meets requirements](https://cloud.google.com/storage/docs/buckets#naming) and is globally unique. We suggest prepending the `<PROJECT_ID>` of the GCP project associated with your workspace, which can be found on the to a descriptive string (e.g., `<PROJECT_ID>_output_data`).\n",
+    "- `Description`: This is description shown in the Resources tab of your workspace's page in Verily Workbench and when the `terra resource list` command is run. The description you provide should add helpful context about the purpose and/or contents of your bucket.\n",
+    "- `Name`: This is the name which can be referenced in Workbench CLI commands so the value should be brief and memorable while communicating the purpose and/or contents of the bucket (e.g. `<STUDY_NAME>-data-bucket`).\n",
+    "- `Object Name`: This is the name of the object in the Google Cloud Storage bucket.\n",
+    "- `Path`:\n",
```

Comment:
I think you need to adjust the spacing here, I see this: https://screenshot.googleplex.com/BTtCezJxPLfYTrq

Reply:
fixed